src/ducks/balance/AccountsList.jsx

```diff
@@ -43,9 +43,14 @@ export const AccountsList = props => {
     [filterByDoc, router]
   )
 
+  const sortedAndFilteredAccounts = useMemo(
+    () => getSortedAccounts(group, accounts).filter(a => a.status !== 'done'),
```

Comment:
The `status` is an attribute of the `io.cozy.bank.accounts` ? I am a bit surprised by this.

Reply:
There is a status when importing accounts. Some accounts are in done, I guess they are accounts that have been added and then deleted.